src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1142,6 +1142,9 @@ public void watchFiles(File buildFile, File outputDirectory, File testOutputDire
                             registerAll(resourceDir.getCanonicalFile().toPath(), watcher);
                             resourceMap.put(resourceDir, true);
                         }
+                    } else if (resourceMap.get(resourceDir) && !resourceDir.exists()) {
```

Comment:
This doesn't need the `resourceMap.get(resourceDir)` condition since it's implied by the `else` clause.